build/templates/grpc_session_options.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from enum import IntEnum
+
+
+class SessionInitializationBehavior(IntEnum):
+    UNSPECIFIED = 0
```

Comment:
what does this really do?
Is this "attach if exists, otherwise initialize"?

Reply:
Yes that's right. I had copied the terms from gRPC, and that term is required/meaningful there. But we should probably choose a better term for our API.

Reply:
Improved per our discussion.